virttest/qemu_monitor.py

```diff
@@ -2214,7 +2214,12 @@ def send_args_cmd(self, cmdlines, timeout=CMD_TIMEOUT, convert=True):
                     value = "=".join(opt[1:])
                     try:
                         if re.match("^[0-9]+$", value):
-                            value = int(value)
+                            # when force convert type to int,exclude 'fd' as bz
+                            # 1853538. And the int type convert is not absolute
+                            # accurate for other values, if hit problem please
+                            # check the expect type of qemu and update.
+                            if not (opt[0] == 'fd'):
+                                value = int(value)
```

Comment:
How about? 
```
                        if opt[0] == "cert-subject":
                            value = value.replace('/', ',')
                        if opt[0] == 'fd':
                            value = str(value)
```
since the previous codes,  ` if re.match("^[0-9]+$", value)` deal with the general situation, the specific situation is moved bellow them, such as `if opt[0] == "cert-subject":` so I think that `fd` option should be handled same as `cert-subject` to keep the previous code design.

Thanks.

Reply:
@YongxueHong  Eh~  the solution above will cause `fd` go into two blocker.
First,  `value = int(value)` as it is all digital, then `str(value)` ad it is `fd`.
 
I think it is a little redundant.  I prefer to exclude `fd` when it is all digital. Thanks!

Reply:
@YongxueHong  Do you have other comment for this issue ?

Reply:
> @YongxueHong Eh~ the solution above will cause `fd` go into two blocker.
> First, `value = int(value)` as it is all digital, then `value = str(value)` as it is `fd`.
> 
> I think it is a little redundant. I prefer to exclude `fd` when it is all digital. Thanks!


If you think it is a redundant, how about` "cert-subject" `? Is the` if opt[0] == "cert-subject":` a redundant? why didn't put 
```
                        if opt[0] == "cert-subject":
                            value = value.replace('/', ',')
```
 into `if re.match("^[0-9]+$", value):` blocker ? and `"cert-subject"` also goes into two blockker.
So I think we should keep consistency.
Either keep the previous code design or keep your present code design.

Reply:
@YongxueHong  That is because i did not check the specific value of `cert-subject` before which is not related with this patch ~

I just checked, it is not all digital, so can not move into the `if re.match("^[0-9]+$", value):` blocker, so ~
Could you explain where you get it can be moved in to the all digital blocker ?

Reply:
> @YongxueHong That is because i did not check the specific value of `cert-subject` before which is not related with this patch ~
> 
> I just checked, it is not all digital, so can not move into the `if re.match("^[0-9]+$", value):` blocker, so ~
> Could you explain where you get it can be moved in to the all digital blocker ?

if `opt[0] == 'fd' `, I think the upper user must pass a string like '12334', so it will re.match("^[0-9]+$", value) when the `opt[0] == 'fd'`

Reply:
`cert-subject` is something include digital, letter and other character such as `=`, `/`. It does not belong to any if block before it, so only one time ~

Here is an example: `spice_x509_server_subj = /C=CZ/L=BRNO/O=SPICE/CN=my Server`
```
cert_s = clone.spice_options.get("spice_x509_server_subj",  "")                            
cert_subj = "%s" % cert_s[1:]
cert_subj += host_ip
cert_subj = "\"%s\"" % cert_subj
```

Reply:
> `cert-subject` is something include digital, letter and other character such as `=`, `/`. It does not belong to any if block before it, so only one time ~
> 
> Here is an example: `spice_x509_server_subj = /C=CZ/L=BRNO/O=SPICE/CN=my Server`
> 
> ```
> cert_s = clone.spice_options.get("spice_x509_server_subj",  "")                            
> cert_subj = "%s" % cert_s[1:]
> cert_subj += host_ip
> cert_subj = "\"%s\"" % cert_subj
> ```

Yes, but it belongs to else blocker:
```
                        else:
                            value = value.strip()
                        if opt[0] == "cert-subject":
                            value = value.replace('/', ',')
```
According to your design, this place should be 
```
                        else:
                            value = value.strip()
                            if opt[0] == "cert-subject":
                                value = value.replace('/', ',')
```
Although I don't have an ideat about the reason why it did not do that.